buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+plugins {
```

Comment:
What's the point of another build.gradle.kts file? I'm not extremely familiar with the gradle systems, including the Kotlin DSL, so I'm just curious.

Reply:
It's necessary for the `buildSrc` module.

Reply:
What is the point of the buildSrc module?

Reply:
Right here, I'm mostly using it to better help organize our dependencies and versions.